debvader/batch_generator.py

```diff
@@ -24,21 +26,27 @@ def __init__(
         y_col_name: column name of data to be fed as target to the network
         batch_size: sample sixe for each batch
         num_iterations_per_epoch: number of samples (of size = batch_size) to be drawn from the sample
-        normalization_func: function to be used for normalization
+        normalization: object of Debvader.normalize.Normalize, used to perform norm and denorm operations
         channel_last: boolean to indicate if the the clast channel corresponds to differnet bands of the input data.
         """
         self.list_of_samples = list_of_samples
         self.x_col_name = x_col_name
         self.y_col_name = y_col_name
         self.batch_size = batch_size
         self.num_iterations_per_epoch = num_iterations_per_epoch
-        self.normalization_func = normalization_func
+        if normalization is not None:
```

Comment:
If you make that check, then I guess the default value for `normalization` should be `None`

Reply:
Yes, by default there was no normalization while training the dc2 simulations

Reply:
I mean in the `__init__`, it should be a keyword with default value `None`